scripts/withdraw.ts

```diff
@@ -0,0 +1,89 @@
+import { BigNumber, BigNumberish } from 'ethers';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { parseJsonFile } from './utils';
+import type { PartnerConfig } from './changePartnerShares';
+
+type MinimumErc20TokenContract = {
+  balanceOf: (address: string) => Promise<BigNumber>;
+};
+
+const printStatus = async (
+  collectorAddress: string,
+  partners: string[],
+  erc20TokenInstance: MinimumErc20TokenContract
+) => {
+  const collectorBalance = await erc20TokenInstance.balanceOf(collectorAddress);
+
+  console.log(`Collector balance: ${collectorBalance.toNumber()}`);
+  for (const partner of partners) {
+    const balance = await erc20TokenInstance.balanceOf(partner);
+    console.log(`Address ${partner} balance: ${balance.toNumber()}`);
+  }
+};
+
+const defaultTxGas = 200000;
```

Comment:
I suggest using the convention for calling this kind of constants. Something like DEFAULT_TX_GAS.

Reply:
done